app/main/services/search_service.py

```diff
@@ -141,11 +142,14 @@ def url_for_search(**kwargs):
             "meta": results['meta'],
             "documents": results['documents'],
             "links": generate_pagination_links(
-                query_args, results['meta']['total'],
+                query_args, results['meta']['total']['value'],
                 page_size, url_for_search
             ),
         }
 
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#hits-total-now-object-search-response
+        response["meta"]["total"] = response["meta"]["total"]["value"]
```

Comment:
Seems like we are mutating this to recreate the old style? But also reading the new value in line 145?
Are there parts of the frontend which are depending on this structure. Could 
we have a TODO for the frontend to read the new structure in future?

Reply:
I think there will be a lot of tech debt out of these changes, so having some TODOs is a probably a good idea.

Not sure what the value of having the frontend reading the new structure is, but I guess we can work it out later.

Reply:
> Are there parts of the frontend which are depending on this structure. 

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/main/views/g_cloud.py#L302